src/jdk.httpserver/share/classes/module-info.java

```diff
@@ -27,6 +27,12 @@
  * Defines the JDK-specific HTTP server API, and provides the jwebserver tool
  * for running a minimal HTTP server.
  *
+ * <p>The API, SPI and the server implementation in this module is intended
+ * for simple usages like local testing, development, and debugging.
+ * Accordingly, the design and implementation of the server is explicitly
```

Comment:
Change "is explicitly" to "are explicitly"?

Reply:
Done